.buildkite/pipeline.yml

```
@@ -14,7 +14,10 @@ steps:
     agents:
       provider: gcp
     plugins:
-      - *google_oidc_plugin
+      - elastic/oblt-google-auth#${BUILDKITE_COMMIT}:
+        lifetime: 10800 # seconds
+        project-id: "elastic-observability-ci"
+        project-number: "911195782929"
```

I think indenting is needed here:


```suggestion
          lifetime: 10800 # seconds
          project-id: "elastic-observability-ci"
          project-number: "911195782929"
```

See https://buildkite.com/docs/pipelines/integrations/plugins/using#adding-a-plugin-to-your-pipeline